File: 구름 알고리즘 미션/기초 구현/합 계산기/합 계산기.js
const readline = require('readline');

(async () => {
	let rl = readline.createInterface({ input: process.stdin });
	let input = [];
	
	for await (const line of rl) {
		input.push(line);
		if(input.length === input[0]+1) rl.close();
	}
	
	const T = input[0];
	let result = 0;
	
	for(let i = 1 ; i <= T ; i++){
		const [a, op, b] = input[i].split(" ");
		switch(op){
			case '+':
				result += +a + +b;
				break;
			case '-':
				result += +a - +b;
				break;
			case '*':
				result += +a * +b;
				break;
			case '/':
				result += Math.floor(+a / +b);
				break;
		}
	}
	console.log(result);
	process.exit();
})();

File: 구름 알고리즘 미션/기초 구현/합 계산기/합 계산기.py
T = int(input())

result = 0
for _ in range(T):
	num1, oper, num2 = map(str, input().split(" "))
	if oper == '+':
		result += int(num1) + int(num2)
	elif oper == '-':
		result += int(num1) - int(num2)
	elif oper == '*':
		result += int(num1) * int(num2)
	else:
		result += int(num1) // int(num2)

print(result)